src/backend/joanie/core/serializers/admin.py

```diff
@@ -168,13 +168,17 @@ class AdminProductSerializer(serializers.ModelSerializer):
     """Serializer for Product model."""
 
     title = serializers.CharField()
-    description = serializers.CharField(required=False)
+    description = serializers.CharField(
+        allow_blank=True, trim_whitespace=False, required=False
+    )
```

Comment:
Is it needed to declare them in the serializer because they are managed by parler in the model ?

Reply:
Yup if we do not declare those fields, the serializer works fine to serialize data but not to deserialize during CUD operations.

Reply:
Did you try to se the

Reply:
In the django parler documentation, I just see that there is a module to make the join between parler and DRF : https://django-parler.readthedocs.io/en/stable/advanced/rest-framework.html

Dos this solution has been discuss already ?

Reply:
So I'll take a look on django-parler-rest and this does not fulfill our needs.

Indeed, this package aims to bind all translations available for each translated fields into serializer data. 

```python
from rest_framework import serializers
from parler_rest.serializers import TranslatableModelSerializer, TranslatedFieldsField
from .models import Country  # Example model


class CountrySerializer(TranslatableModelSerializer):
    translations = TranslatedFieldsField(shared_model=Country)

    class Meta:
        model = Country
        fields = ('id', 'country_code', 'translations')
```

```json
{
    "id": 528,
    "country_code": "NL",
    "translations": {
        "nl": {
            "name": "Nederland",
            "url": "http://nl.wikipedia.org/wiki/Nederland"
        },
        "en": {
            "name": "Netherlands",
            "url": "http://en.wikipedia.org/wiki/Netherlands"
        },
        "de": {
            "name": "Niederlande",
            "url": "http://de.wikipedia.org/wiki/Niederlande"
        }
    }
}
```